Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -249,17 +249,23 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testCallToGetCustomerInfoWithPendingTransactionsPostsReceiptOnlyOnce() async throws {
+        // forceRenewalOfSubscription doesn't work well, so we use this instead
+        setShortestTestSessionTimeRate(self.testSession)
+
         // This test requires the "production" behavior to make sure
         // we don't refresh the receipt a second time when posting the second transaction.
         self.enableReceiptFetchRetry = false
 
         self.serverDown()
 
-        try await self.purchaseMonthlyProduct(allowOfflineEntitlements: true)
+        try await self.purchaseShortestDuration(allowOfflineEntitlements: true)
         try self.testSession.forceRenewalOfSubscription(
-            productIdentifier: await self.monthlyPackage.storeProduct.productIdentifier
+            productIdentifier: await self.shortestDurationProduct.productIdentifier
```

Comment:
This isn't part of the changes performed in the original PR: https://github.com/RevenueCat/purchases-ios/pull/3833/files#diff-87ccf0103b9d7e4aca89cdbc663aea182648e269122559c1f92e436a52a7fe16R245.

But this test was failing for me without forcing the renewal here.